UI/NpcList/NpcList.gd

```diff
@@ -0,0 +1,26 @@
+extends Control
+
+onready var npcRow = load("res://UI/NpcList/NPCRow.tscn")
+onready var container = get_node("VBoxContainer/ItemList/NpcListControld/ScrollContainer/HBoxContainer")
```

Comment:
onready var container = $VBoxContainer/ItemList/NpcListControld/ScrollContainer/HBoxContainer

Reply:
It's the same basically just a bit shorter? But ok.

Reply:
it works the same but I use $ everywhere. It's shorter too yeah ^^